packages/app/provider/navigation/index.tsx

```diff
@@ -22,7 +22,12 @@ export function NavigationProvider({ children }: { children: ReactNode }) {
               "sign-up": "sign-up",
               dashboard: {
                 screens: {
-                  search: "dashboard/search",
+                  search: {
+                    screens: {
+                      "search-for": "dashboard/search",
+                      beatmaker: "dashboard/beatmaker/:id",
```

Comment:
Maybe I should change link from `dashboard/beatmaker/:id` to `dashboard/search/beatmaker/:id`

Reply:
yes